File: Basic/Calculate Nth Fibonacci Number/SolutionByShraman.py
def fibonacci(n):
    if n == 1:
        return 0
    if n == 2:
        return 1
    return fibonacci(n-1)+fibonacci(n-2)


print("N = ", end="")
N = int(input())
N = fibonacci(N)
print(f"Output = {N}")

File: Basic/Check if A String Contains Only Digits/SolutionByShraman.cpp
#include <iostream>
#include <ctype.h>
using namespace std;

string check(string str)
{
    char ch;
    int l = str.length();
    for (int i = 0; i < l; i++)
    {
        ch = str.at(i);
        if (isdigit(ch) == 0)
            return "No";
    }
    return "Yes";
}

int main()
{
    string str;
    cout << "Input: ";
    cin >> str;
    cout << "Output: " << check(str);
}
